leaf-ui/js/onFunctions.js

```diff
@@ -526,21 +541,28 @@ function switchToModellingMode() {
     // and clears results if model changed during modelling mode
     // previousModel is NOT of type Model
     previousModel = JSON.parse(JSON.stringify(model));
+    
+    $('#on-off').css("display", "");
```

Comment:
If we are re-sectioning these functions into appear and disappear, why is this in a separate section?